cmd/shfmt/shfmt.1.scd

```diff
@@ -73,6 +73,9 @@ predictable. Some aspects of the format can be configured via printer flags.
 *-filename* str
 	Provide a name for the standard input file.
 
+	Use of this flag is necessary for EditorConfig support to work,
```

Comment:
.."for EditorConfig support to work with standard input,"

I think that makes it a little clearer in this standalone sentence that this isn't necessary when a file path is provided as the last parameter.

Reply:
Hm, I actually had it written like that first, but the section already begins with "for the standard input file", so I thought it would be redundant. I agree that it's better to be explicit, on second thought.